content/en/docs/cluster-administration/nodes.md

```diff
@@ -1,38 +1,76 @@
+## Prerequisites
 
-![Node Status](/images/docs/cluster-administration/node_status.png)
+You need an account granted a role including the authorization of **Clusters Management**. For example, you can log in the console as `admin` directly or create a new role with the authorization and assign it to an account.
 
-- **Status** : Node current status, indicate node is available or not.  
-- **CPU** : Node current cpu usage, these values are real-time numbers.  
-- **Memory** : Current memory usage, same with _CPU_ stats, are also real-time numbers.  
-- **Allocated CPU** : Calculated from summing up all pods cpu requests on this node, means how much amount of CPU reserved for workloads on this node, even workloads are using less CPU resource. This metric is vital to kubernetes scheduler, kube-scheduler will favor lower _Allocated CPU_ usage nodes when scheduling a pod in most cases. For more details, refer to [manage resources containers](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/).  
-- **Allocated Memory** : Calculated from summing up all pods memory requests on this node. Same with _Allocated CPU_.
+## Node Status
 
-> **Note:** _CPU_ and _Allocated CPU_ are differ in most of times, so is memory, this is normal. As a cluster administrator, should focus on both kind of metrics instead of just one. It's always a good practice to set resources requests and limits for each pod to match the their real usage. Over allocating can lead to low cluster utilization, otherwise can lead to high pressure on cluster, and even cluster unhealthy.
+Cluster nodes are only accessible to cluster administrators. Some node metrics are very important to clusters. Therefore, it is the administrator's responsibility to watch over these numbers and make sure nodes are available. Follow the steps below to view node status.
 
-## Nodes Management
+1. Click **Platform** in the top left corner and select **Clusters Management**.
 
-![Node Detail](/images/docs/cluster-administration/node_detail.png)
+![clusters-management-select](/images/docs/cluster-administration/node-management/clusters-management-select.jpg)
 
-- **Cordon/Uncordon** : Marking a node as unschedulable are very useful during a node reboot or othere maintenance. The kubernetes scheduler will not schedule new pods to this node if it's been marked unschedulable, but does not affect existing workloads already on this Node. In KubeSphere, you mark a node as unschedulable by click button _Cordon_ in node detail page, node will be schedulable again if click the button again.
+2. If you have enabled the [multi-cluster feature](http://localhost:1313/docs/multicluster-management) with member clusters imported, you can select a specific cluster to view its nodes. If you have not enabled the feature, refer to the next step directly.
```

Comment:
The link contains a localhost address `http://localhost:1313/`, please update it.